cmfpy/algs/accelerated.py

```diff
@@ -0,0 +1,73 @@
+import numpy.linalg as la
+from .base import AbstractOptimizer
+
+
+class AcceleratedOptimizer(AbstractOptimizer):
+    """
+    Accelerated Update Rules.
+
+    From Gillis, https://arxiv.org/abs/1107.5194v2.
+    """
+
+    def __init__(self, data, dims, patience=3, tol=1-5,
+                 max_inner=0, inner_thresh=0, weightW=1, weightH=1,
+                 **kwargs):
+        super().__init__(data, dims, patience=patience, tol=tol, **kwargs)
+
+        self.max_inner = max_inner
+        self.inner_thresh = inner_thresh
+        self.weightW = weightW
+        self.weightH = weightH
```

Comment:
Can you define what all of these parameters (max_iter, inner_thres, weightW, weightH) mean in a docstring for `__init__`?